### ISSUE TITLE
You can auto-attack the ram from inside keep door

### ISSUE THREAD

**Issue by rekkir** [WarEmu/WarBugs#10242]

**Comment by Tazo83** [comment]
yes i can confirm this i was inside the keep hitting the ram and with WB ram can be killed easy while staying inside the keep 

https://clips.twitch.tv/YummyCooperativePieWoofer
